src/main/java/net/minecraftforge/fml/server/FMLServerHandler.java

```diff
@@ -96,6 +100,17 @@ private FMLServerHandler()
     public void beginServerLoading(MinecraftServer minecraftServer)
     {
         server = minecraftServer;
+        ServerEula eula = new ServerEula(new File("eula.txt"));
```

Comment:
I don't think so as this would create unnecessary patches, and it should not have any impact